beavis-ci.sh

```diff
@@ -121,6 +121,9 @@ while [ $# -gt 0 ]; do
             shift
             kernel="$1"
             ;;
+        *)
+            repo="$1"
+            ;;
     esac
     shift
 done
```

Comment:
After this loop, the script should check if `repo` is not empty, and abort if it is still empty.

Reply:
Good call.

Reply:
OK, if repo is not set, script prints header and exits.